src/components/tools/image-tool.tsx

```diff
@@ -41,30 +41,17 @@ let nextImageToolId = 0;
 @inject("stores")
 @observer
 export default class ImageToolComponent extends BaseComponent<IProps, IState> {
-  public state: IState = { isLoading: true };
-
+  public state: IState = { isLoading: true,
+                           imageContentUrl: this.getContent().url
```

Comment:
I wasn't sure whether `getContent()` is guaranteed to exist at the time of public instance member initialization, but having looked it up, I think it is.